modules/fts/pages/fts-consistency-level.adoc

```diff
@@ -0,0 +1,20 @@
+= Consistency Level
+
+#What is consistency level? Need Information#
```

Comment:
Missing information

Reply:
There's documentation on consistency level already that needs to here. It can take any of these 2 options - https://docs.couchbase.com/server/current/fts/fts-queries.html#scan-consistency

Reply:
@abhinavdangeti Yes, we took the information about two parameters. But, what exactly 'level' attribute indicates? is this true: "the minimum number of vectors that must acknowledge a read or write operation before the operation can be considered as successful"?